packages/js/product-editor/tsconfig.json

```diff
@@ -5,6 +5,12 @@
 		"outDir": "build-module",
 		"declaration": true,
 		"declarationMap": true,
-		"declarationDir": "./build-types"
-	}
+		"declarationDir": "./build-types",
+		"resolveJsonModule": true
+	},
+	"include": [
+		"**/*.d.ts",
```

Comment:
Do you need this `"**/*.d.ts"` line in addition to the one below? BTW I also made these changes in my [name block PR](https://github.com/woocommerce/woocommerce/pull/37132) so we many have a couple conflicts.

Reply:
I found that without including this, the typings did not make it in and would throw an error on build.

Reply:
Interesting, didn't see any build errors with mine. Sounds good 👍🏻